x/ccv/provider/keeper/genesis.go

```diff
@@ -158,7 +158,7 @@ func (k Keeper) ExportGenesis(ctx sdk.Context) *types.GenesisState {
 		k.GetAllValsetUpdateBlockHeights(ctx),
 		consumerStates,
 		k.GetAllUnbondingOps(ctx),
-		&ccv.MaturedUnbondingOps{Ids: k.GetMaturedUnbondingOps(ctx)},
+		&types.MaturedUnbondingOps{Ids: k.GetMaturedUnbondingOps(ctx)},
```

Comment:
I fully agree that we should use the `ccvtypes` alias everywhere, instead of `types` or `ccv`. However I don't think that sort of change is appropriate for this PR as we're already at 4k lines of changes. 

Making the alias change in this file would add ~14 lines from the look of it, but other files have even more refs to the undesirable aliases